xml/virt_support.xml

```diff
@@ -5,175 +5,209 @@
+   The total number of virtual CPUs should be proportional to the number of available physical CPUs.	 
+  </para>
+  <note>
+   <title>32-bit hypervisor</title>
+   <para>
+    With &productname; 11 SP2, we removed the 32-bit hypervisor as a virtualization host.
+    32-bit virtual guests are not affected and are fully supported with the provided 64-bit hypervisor.    
+   </para>
+  </note>
+  <sect2 xml:id="virt-kvm-limits">
+   <title>KVM Limits</title>
+   <para>
+    Supported (and tested) virtualization limits of a &productname; &productnumber; host running Linux guests on x86-64. For other operating systems, refer to the specific vendor.
+   </para>
+   <itemizedlist>
+    <title>Virtual Machine Limits</title>
+    <listitem><para>Maximum virtual CPUs per VM: 288</para></listitem>
+    <listitem><para>Maximum memory per VM: 4 TiB</para></listitem>
+   </itemizedlist>
+   <itemizedlist>
+    <title>Host Limits</title>
+    <listitem><para>Other limits are identical to those of &productname;.</para></listitem>
+   </itemizedlist>
+  </sect2>
+  <sect2 xml:id="virt-xen-limits">
+   <title>Xen Limits</title>
+   <itemizedlist>
+    <title>Virtual Machine Limits</title>
+    <listitem><para>Maximum virtual CPUs per VM: 128 (HVM), 64 (HVM Windows guest) or 512 (PV)</para></listitem>
+    <listitem><para>Maximum memory per VM: 2 TB (64bit guest), 16 GB (32-bit guest with PAE)</para></listitem>
+   </itemizedlist>
+   <itemizedlist>
+    <title>Host Limits</title>
+    <listitem><para>Maximum total physical CPUs: 1024</para></listitem>
+    <listitem><para>Maximum physical memory: 16 TB</para></listitem>
+   </itemizedlist>
+  </sect2>
+ </sect1>
+ 
+ <sect1 xml:id="virt-support-hosts">
+  <title>Supported Host Environments (Hypervisors)</title>
+  <para>
+   Support status of &productname; &productnumber; running as a guest operating system on top of various virtualization hosts (hypervisors).
+  </para>
   <itemizedlist>
-   <title>The following third party host environments are supported:</title>
-   <listitem>
-    <para>
-     VMware ESXi 6.5, 6.7
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Microsoft Windows 2008 R2 SP1+, 2012+, 2012 R2+, 2016, 2019
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Citrix XenServer 7.0, 7.1, 8.0
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Oracle VM 3.4
-    </para>
-   </listitem>
-   <listitem>
-    <para>
-     Nutanix Acropolis Hypervisor with AOS 5.8
-    </para>
-   </listitem>
+   <title>The following &suse; host environments are supported</title>
+   <listitem><para>&sls; 11 SP4: Xen and KVM</para></listitem>
+   <listitem><para>&sls; 12 SP1 to SP5: Xen and KVM</para></listitem>
+   <listitem><para>&sls; 15 GA to SP3: Xen and KVM</para></listitem>
+<!--   <listitem><para>SLES 15 SP1 to SP3: WSL 2</para></listitem> -->
```

Comment:
I personally compiled that list and it took a lot of doing!

Do we no longer support 3rd party host OSes at all?! I am shocked if so. If we do, we need to specify them.

Reply:
There is the list of third party hypervisors.